frontend-react/src/contexts/SessionContext.tsx

```diff
@@ -49,16 +50,20 @@ const SessionProvider = ({
         return activeMembership?.memberType === MemberType.PRIME_ADMIN;
     }, [activeMembership?.memberType]);
 
+    const context = useMemo(
```

Comment:
Minor point, but since this entire Context value is going to be memoized, would you mind removing the `useMemo` for `isAdminStrictCheck` above?  Since we're no longer allowing users to toggle between different `member_type` values, we can assume that any changes to `activeMembership` will result in a reference change so `isAdminStrictCheck` would be recomputed anyway.  (Plus, I thiiiiink the  old `member_type` change feature was just spreading on to a new object so the reference would change anyway 🤷)